### PR TITLE
Boot up staging env

### PR DESCRIPTION
**Pull request by ismail-shahzad**
This PR boots up a staging environment for WRI ODP

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@ name: Build and Deploy to AWS
 on:
   push:
     branches:
-      - dev
+      - init-staging
```
Comment (luccasmmg) [inline .github/workflows/main.yml]: Shouldnt this be "staging" instead of "init-staging"

File: ckan-deployment/frontend/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+FROM node:18
+
+# Create app directory
+WORKDIR /usr/src/app
+
+ENV NEXTAUTH_SECRET "secret"
+ENV NEXTAUTH_URL "http://localhost:3000" 
+ENV CKAN_URL "https://demo.dev.datopian.com"
```
Comment (luccasmmg) [inline ckan-deployment/frontend/Dockerfile:8]: I've left these env variables hard coded but they need to be changed, are they? Right now the frontend will call the bizdev instance, which i assume is not ideal

**Review by mpolidori — APPROVED** [review]
Looks good!

**Review by dmannarino — APPROVED** [review]
👍

File: ckan-deployment/frontend/Dockerfile (hunk 2 above)
Comment (ismail-shahzad) [reply ckan-deployment/frontend/Dockerfile:8]: Yes we will change them accordingly once we add a CI/CD for frontend

File: .github/workflows/main.yml (hunk 1 above)
Comment (ismail-shahzad) [reply .github/workflows/main.yml]: I think this is showing old commit
It was already fixed in https://github.com/wri/wri-odp/pull/21/commits/d9555d1591b2f396cafcb4c8dc307e436881e80a

Can be seen in the merged code https://github.com/wri/wri-odp/blob/staging/.github/workflows/main.yml#L5

**Pull request merged**